website/docs/r/mysql_server.html.markdown

```diff
@@ -86,9 +75,52 @@ Possible values for `storage_mb` when using a SKU Name of `Standard` are:
 
 * `sku` supports the following:
 
-* `name` - (Optional) Specifies the SKU Name for this MySQL Server. Possible values are: `MYSQLB50`, `MYSQLB100`, `MYSQLS100`, `MYSQLS200`, `MYSQLS400` and `MYSQLS800`.
-* `capacity` - (Optional) Specifies the DTU's for this MySQL Server. Possible values are `50` and `100` DTU's when using a `Basic` SKU and `100`, `200`, `400` or `800` when using the `Standard` SKU.
-* `tier` - (Optional) Specifies the SKU Tier for this MySQL Server. Possible values are `Basic` and `Standard`.
+* `name` - (Required) Specifies the SKU Name for this MySQL Server. See the `name` values by `tier` and `capacity` table below for valid values.
```

Comment:
since there's a much larger set now I'm going to suggest we remove this in favour of linking to the product page for more info - as such can we remove "See the `name` values by `tier` and `capacity` table below for valid values."?